<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget_2{background-color:rgba(37, 39, 85, 200);
border-radius: 20px;}</string>
    </property>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>321</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget_3{background-color:rgba(37, 39, 85, 100);
border-radius: 20px;
border:3px solid;}</string>
     </property>
     <widget class="QLabel" name="login_error">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 14, 30);
font: 7pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="accountbtn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>260</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#accountbtn{
background-color: qlineargradient(spread:pad, x1:0.017, y1:0.937409, x2:1, y2:0, stop:0 rgba(17, 9, 61, 178), stop:1 rgba(255, 255, 255, 68));
border-radius: 5px;
color: rgba(226, 226, 226, 227);
border: 1px solid rgba(0,0,0,0.15);
}
QPushButton#accountbtn:hover{
background-color: qlineargradient(spread:pad, x1:0.017, y1:0.937409, x2:1, y2:0, stop:0 rgba(17, 9, 61, 178), stop:0.295455 rgba(255, 255, 255, 68));
}
QPushButton#accountbtn:pressed{
padding-left:2px;
padding-top:2px;
background-color:rgb(100, 100, 132);
}</string>
      </property>
      <property name="text">
       <string>Utwórz konto</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginbtn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#loginbtn{
background-color: qlineargradient(spread:pad, x1:0.017, y1:0.937409, x2:1, y2:0, stop:0 rgba(17, 9, 61, 178), stop:1 rgba(255, 255, 255, 68));
border-radius: 5px;
color: rgba(226, 226, 226, 227);
border: 1px solid rgba(0,0,0,0.15);
}
QPushButton#loginbtn:hover{
background-color: qlineargradient(spread:pad, x1:0.017, y1:0.937409, x2:1, y2:0, stop:0 rgba(17, 9, 61, 178), stop:0.295455 rgba(255, 255, 255, 68));
}
QPushButton#loginbtn:pressed{
padding-left:2px;
padding-top:2px;
background-color:rgb(100, 100, 132);
}</string>
      </property>
      <property name="text">
       <string>Zaloguj</string>
      </property>
     </widget>
     <widget class="QLabel" name="logintxt">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#welcometxt{font: 24pt &quot;MS Shell Dlg 2&quot; ;
font-weight: bold;}</string>
      </property>
      <property name="text">
       <string>Logowanie</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="infotxt">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(197, 197, 197);</string>
      </property>
      <property name="text">
       <string>Zaloguj się do istniejącego konta</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="login_field">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom: 2px solid rgba(105,118, 132, 255);
color: rgba(0,0,0,230);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_field">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom: 2px solid rgba(105,118, 132, 255);
color: rgba(0,0,0,230);</string>
      </property>
      <property name="placeholderText">
       <string>Hasło</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>300</y>
        <width>151</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(background.png)</string>
      </property>
     </widget>
     <widget class="QLabel" name="forgot_passwordtxt">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>240</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#forgot_passwordtxt{font: 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(197, 197, 197);
}
QLabel#forgot_passwordtxt:hover{
color:white;}</string>
      </property>
      <property name="text">
       <string>Zapomniałeś hasła?</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>accountbtn</zorder>
     <zorder>loginbtn</zorder>
     <zorder>login_error</zorder>
     <zorder>logintxt</zorder>
     <zorder>infotxt</zorder>
     <zorder>login_field</zorder>
     <zorder>password_field</zorder>
     <zorder>widget</zorder>
     <zorder>forgot_passwordtxt</zorder>
    </widget>
    <widget class="QPushButton" name="exitbtn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>0</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#exitbtn{
background-color: qlineargradient(spread:pad, x1:0.017, y1:0.937409, x2:1, y2:0, stop:0 rgba(17, 9, 61, 0), stop:1 rgba(255, 255, 255, 0));
border-radius: 7px;
color: rgba(226, 226, 226, 227);
font-weight: bold;
}
QPushButton#exitbtn:hover{
background-color: qlineargradient(spread:pad, x1:0.017, y1:0.937409, x2:1, y2:0, stop:0 rgba(17, 9, 61, 200), stop:0.295455 rgba(255, 255, 255, 0));
}
QPushButton#exitbtn:pressed{
padding-left:2px;
padding-top:2px;
background-color:qlineargradient(spread:pad, x1:0.017, y1:0.937409, x2:1, y2:0, stop:0 rgba(17, 9, 61, 200), stop:0.295455 rgba(255, 255, 255, 0));
}</string>
     </property>
     <property name="text">
      <string>Х</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>login_field</tabstop>
  <tabstop>password_field</tabstop>
  <tabstop>loginbtn</tabstop>
  <tabstop>accountbtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
